prowler/providers/kubernetes/services/core/core_service.py

```diff
@@ -0,0 +1,83 @@
+from typing import List, Optional
+
+from kubernetes import client
+from pydantic import BaseModel
+
+from prowler.lib.logger import logger
+from prowler.providers.kubernetes.lib.service.service import KubernetesService
+
+
+################## Core ##################
+class Core(KubernetesService):
+    def __init__(self, audit_info):
+        super().__init__(audit_info)
+        self.client = client.CoreV1Api(self.api_client)
+
+        self.pods = []
```

Comment:
`pods` should be a map, remember that we always do lookups in this objects, you can use the pod UID as the key to be unique in the map.

Reply:
Done